src/Amadevus.RecordGenerator.Attributes/Features.cs

```diff
@@ -42,16 +42,44 @@ public enum Features
         /// <c>Deconstruct</c> method:
         /// <see href="https://docs.microsoft.com/pl-pl/dotnet/csharp/deconstruct#deconstructing-user-defined-types"/>
         /// </summary>
-        Deconstruct = 0b_10000,
+        Deconstruct = 0b_1000_0,
```

Comment:
Not the formatting I thought about. I'd expect left-aligned breaks, e.g.
* `0b_1_000`
* `0b_10_000`
* `0b_100_000`
* `0b_1000_000`